.github/workflows/sample.yml

```diff
@@ -34,3 +34,4 @@ jobs:
         resource-group: rg-hoge
         name: ctapp-hoge
         image: mcr.microsoft.com/azuredocs/containerapps-helloworld:latest
+        commit-hash: ${{ github.sha }}
```

Comment:
Why do you separate `image` and `commit-hash` ?
Isn't an `image` enough? ex `mcr.microsoft.com/azuredocs/containerapps-helloworld:${{ github.sha }}`

Reply:
> Isn't an image enough? ex `mcr.microsoft.com/azuredocs/containerapps-helloworld:${{ github.sha }}`

I think It depends on image-build-push step which string is set as a tag part of image name. Can we assume that `${{ github.sha }}` is always used as image tag?

Reply:
I see.  If `commit-hash` is `docker tag`, I recommend changing the parameter name. ex `tag: ${{ github.sha }}`

Reply:
I misread. This tag is used as `revision name`.  So, we should change the name.

Reply:
I suppose it's possible to get github.sha on typescript side.

```typescript
import * as github from "@actions/github";
github.context.sha
```

I'm not sure if it actually works, but it seems to be a right way to get hash.
Please check the link below.
https://github.com/actions/toolkit/blob/main/packages/github/src/context.ts